assets/chat/js/formatters/UrlFormatter.js

```diff
@@ -71,9 +71,19 @@ export default class UrlFormatter {
       const m = decodedUrl.match(self.linkregex);
       if (m) {
         const encodedUrl = self.encodeUrl(m[0]);
+        const maxUrlLength = 90;
+        let urlText = encodedUrl;
+        if (encodedUrl.length > maxUrlLength) {
```

Comment:
Although it makes no difference behaviorally, this logic has to do with adjusting the URL's text, so it makes more sense to compare the length of `urlText`.

```suggestion
        if (urlText.length > maxUrlLength) {
```

Reply:
Sure will sort